posthog/clickhouse/test/__snapshots__/test_schema.ambr

```diff
@@ -1831,6 +1929,43 @@
   GROUP BY `$session_id`, team_id
   
   
+  '''
+# ---
+# name: test_create_table_query[sharded_app_metrics2]
+  '''
+  
+  CREATE TABLE IF NOT EXISTS sharded_app_metrics2 ON CLUSTER 'posthog'
+  (
+      team_id Int64,
+      timestamp DateTime64(6, 'UTC'),
+      -- The name of the service or product that generated the metrics.
+      -- Examples: plugins, hog
+      app_source LowCardinality(String),
+      -- An id for the app source.
+      -- Set app_source to avoid collision with ids from other app sources if the id generation is not safe.
+      -- Examples: A plugin id, a hog application id
+      app_source_id String,
+      -- A secondary id e.g. for the instance of app_source that generated this metric.
+      -- This may be ommitted if app_source is a singleton.
+      -- Examples: A plugin config id, a hod application config id
+      instance_id String,
+      successes SimpleAggregateFunction(sum, Int64),
+      skipped SimpleAggregateFunction(sum, Int64),
+      failures SimpleAggregateFunction(sum, Int64),
+      -- The UUID of the event that caused this error to be generated.
+      error_uuid UUID,
+      error_type String,
+      error_details String CODEC(ZSTD(3))
+      
+  , _timestamp DateTime
+  , _offset UInt64
+  , _partition UInt64
+  
+  )
+  ENGINE = ReplicatedAggregatingMergeTree('/clickhouse/tables/77f1df52-4b43-11e9-910f-b8ca3a9b9f3e_{shard}/posthog.sharded_app_metrics2', '{replica}')
+  PARTITION BY toYYYYMM(timestamp)
```

Comment:
The previous table had no TTL and we allow querying all of it in the app, but I feel like 12 months TTL would be more than enough?

Reply:
There's a 90 day TTL, it's just disabled in TEST https://github.com/PostHog/posthog/blob/0a9e21d18e3cff0f4db2b1456daa7d13326c2719/posthog/clickhouse/kafka_engine.py#L45-L47